recognition/include/pcl/recognition/impl/implicit_shape_model.hpp

```diff
@@ -404,8 +404,8 @@ pcl::features::ISMModel::saveModelToFile (std::string& file_name)
   for (unsigned int i_cluster = 0; i_cluster < number_of_clusters_; i_cluster++)
   {
     output_file << static_cast<unsigned int> (clusters_[i_cluster].size ()) << " ";
-    for (unsigned int i_visual_word = 0; i_visual_word < static_cast<unsigned int> (clusters_[i_cluster].size ()); i_visual_word++)
-      output_file << clusters_[i_cluster][i_visual_word] << " ";
+    for (const unsigned int &i_visual_word : clusters_[i_cluster])
```

Comment:
Minor: this is not an index variable anymore, so <strike>i_</strike>`visual_word` is more appropriate name.